static/templates/me_message.handlebars

```diff
@@ -1,14 +1,20 @@
-<span class="sender-status">
-    <span class="sender_name-in-status auto-select" role="button" tabindex="0">{{msg/sender_full_name}}</span>
-    {{#if sender_is_bot}}
-    <i class="zulip-icon bot" aria-label="{{t 'Bot' }}"></i>
-    {{/if}}
-
-    <span class="status-message auto-select">
-        {{{ status_message }}}
+<span class="message_sender no-select">
```

Comment:
Hmm, this change puts the entire body of the message inside a `.message_sender` CSS class.  I don't think that's a good idea, structurally.  I think that means the right fix here is to put this block inside the `sender-status` block (and then probably make that a `div` rename that to e.g. `status_message_container` or something else that's clearer about what it is for).  Or possibly just eliminate `sender-status` as a construct; not sure what's right there.